tests/testthat/_snaps/extract.md

```diff
@@ -0,0 +1,94 @@
+# mis-specified extract function
+
+    Code
+      res_extract_warning <- fit_resamples(wf, boots, control = control_resamples(
+        extract = raise_warning))
+    Message
+      ! Bootstrap1: preprocessor 1/1, model 1/1 (extracts): AHHH
+      ! Bootstrap2: preprocessor 1/1, model 1/1 (extracts): AHHH
+      ! Bootstrap3: preprocessor 1/1, model 1/1 (extracts): AHHH
+
+---
+
+    Code
+      res_extract_error <- fit_resamples(wf, boots, control = control_resamples(
+        extract = raise_error))
+    Message
+      x Bootstrap1: preprocessor 1/1, model 1/1 (extracts): Error in extractor(object): AHHH
+      x Bootstrap2: preprocessor 1/1, model 1/1 (extracts): Error in extractor(object): AHHH
+      x Bootstrap3: preprocessor 1/1, model 1/1 (extracts): Error in extractor(object): AHHH
```

Comment:
I'd like to make these `Error in extractor(object)` bits of the error messages more informative, but that would require changes to more general machinery (and many snapshot updates) that would be better off living in a separate PR.